src/lib/comms/include/sol-http-server.h

```diff
@@ -54,6 +55,29 @@ extern "C" {
 struct sol_http_server;
 
 /**
+ * @struct sol_http_server_config
+ * @brief
+ *
+ * It's created with sol_http_server_new() and should be later
+ * deleted with sol_http_server_del()
+ */
+struct sol_http_server_config {
+#ifndef SOL_NO_API_VERSION
+#define SOL_HTTP_SERVER_CONFIG_API_VERSION (1)  /**< compile time API version to be checked during runtime */
+    /**
+     * must match #SOL_HTTP_SERVER_CONFIG_API_VERSION at runtime.
+     */
+    uint16_t api_version;
+#endif
+
+    uint16_t port;
```

Comment:
i do not know, i will check other places doing the same to identify the proper pattern.
I guess SPI is the different.